rotkehlchen/tests/api/test_nfts.py

```diff
@@ -56,7 +56,6 @@ def test_nft_query(rotkehlchen_api_server, start_with_valid_premium):
         if entry['token_identifier'] == '_nft_0xc3f733ca98e0dad0386979eb96fb1722a1a05e69_129':
 
             assert entry['name'] == 'MoonCat #129: 0x0082206dcb'
-            assert entry['permalink'] == 'https://opensea.io/assets/0xc3f733ca98e0dad0386979eb96fb1722a1a05e69/129'  # noqa: E501
```

Comment:
but should still have asserted on the existence of the `permalink` since in production we do use it

Reply:
Sure, will change it

Reply:
keep it mind for another PR.